### ISSUE TITLE
bug: handling of relations/foreign keys in configuration

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eppye-bots/bots#162]
```
when deleting eg a partner that is referenced in other objects (route, 
translation, etc) the reference is set to NULL in these objects.
That can lead to configuration errors:
- a partner-dependent translation will change to a non-partner dependent 
translation. This can lead to using the wrong translation.

In django < 1.3 all references/foreign keys are handled as cascaded deletes. 
That would not be OK. So the cascading deletes where disabled and all foreign 
keys where set to NULL.

In django 1.3 it can be indicated how to handle foreign keys for deletes.

So:
- use django default mechanism to handle foreign keys
- use django default confirmation for deletes & cascaded deletes; some deletes 
are not possible.
- basically bots-specific code to handle this has disappeared ;-)


```

Original issue reported on code.google.com by `hjebb...@gmail.com` on 22 Aug 2012 at 1:32
